magenta/tools/magenta-install.sh

```diff
@@ -85,7 +85,7 @@ echo "setting up magenta environment"
 echo "=============================="
 echo ""
 
-conda create -n magenta python=2.7
+conda create -n magenta python=3.6
```

Comment:
Let's actually move this to python=3.7 since that's what we do our Travis tests with

Reply:
Done.